release/downloadIstioCandidate.sh

```diff
@@ -21,13 +21,13 @@
 # The script fetches the latest Istio release candidate and untars it.
 # You can pass variables on the command line to download a specific version
 # or to override the processor architecture. For example, to download
-# Istio 1.6.8 for the x86_64 architecture,
-# run curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 sh -.
+# Istio 1.6.8 for the x86_64 architecture and linux OS,
+# run curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 TARGET_OS=Linux sh -.
```

Comment:
from L34, this should be `linux`?

Reply:
In fact, Istio can be downloaded correctly here using either "linux" or "Linux". However, considering the difference between TARGET_ARCH and ISTIO_ARCH, the TARGET_OS selection corresponds to the output of uname, while OSEXT in the original text is used together with ISTIO_ARCH to concatenate the file name.

At the same time, this is consistent with the logic at https://github.com/istio/istio/blob/f0f295a2c2366a91ee9c19ffc11326c1a8f48488/release/downloadIstioCandidate.sh#L105 .